src/BenchmarksDriver/Program.cs

```diff
@@ -1325,7 +1337,7 @@ private static Task WriteJobsToSql(ServerJob serverJob, ClientJob clientJob, str
                         string.IsNullOrEmpty(connectionFilter) ? (object)DBNull.Value : connectionFilter);
                     p.AddWithValue("@WebHost", webHost.ToString());
                     p.AddWithValue("@KestrelThreadCount", (object)kestrelThreadCount ?? DBNull.Value);
-                    p.AddWithValue("@ClientThreads", clientThreads);
+                    p.AddWithValue("@ClientThreads", (object)clientThreads);
```

Comment:
I assume that anything that is specific to workers can't be mapped to specific database columns.
But we should have a generic column that records the parameters in case we want to analyze it. Then we can just add an NVARCHAR(max) column that would contain a CSV with the worker properties. CSV would be easier to manipulate than Json in a db column.